src/EditorFeatures/Core.Wpf/Suggestions/SuggestedActionsSource.cs

```diff
@@ -623,6 +623,17 @@ private void OnSuggestedActionsChanged(Workspace currentWorkspace, DocumentId? c
                 }
 
                 var provider = _owner;
+
+                // The _owner flag is cleared on Dispose. This should only occur when cancellation has already been
+                // requested, but at least in 16.8 Preview 3.0 it was not always the case.
+                cancellationToken.ThrowIfCancellationRequested();
+                //Assumes.Present(provider);
```

Comment:
Just so we don't lose the request I had on the other PR: let's get a bug URL in the code pointing to this.